src/Effects/Reverse.h

```diff
@@ -0,0 +1,95 @@
+/*
+  ZynAddSubFX - a software synthesizer
+
+  Reverse.h - Reverse Delay Effect
+  Copyright (C) 2023-2024 Michael Kirchner
+  Author: Michael Kirchner
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+*/
+
+#ifndef REVERSE_H
+#define REVERSE_H
+
+#include "Effect.h"
+
+namespace zyn {
+
+class Reverter;
+
+/**Reverse Effect*/
+class Reverse final:public Effect
+{
+    public:
+        Reverse(EffectParams pars, const AbsTime *time_);
+        ~Reverse();
+        void update();
+        void out(const Stereo<float *> &input);
+        unsigned char getpresetpar(unsigned char npreset, unsigned int npar);
+        void setpreset(unsigned char npreset);
+        /**
+         * Sets the value of the chosen variable
+         *
+         * The possible parameters are:
+         *   -# Volume
+         *   -# Panning (member of parent class)
+         *   -# Delay
+         *   -# Phase
+         *   -# Crossfade
+         *   -# Sync Mode
+         *   -# Stereo
```

Comment:
Stereo should be between phase/delay to be consistent with parameter id ordering, no?